src/Components/Facility/ConsultationDetails/ConsultationUpdatesTab.tsx

```diff
@@ -104,6 +108,21 @@ export const ConsultationUpdatesTab = (props: ConsultationTabProps) => {
     fetchData();
   }, [props.consultationData]);
 
+  useQuery(routes.listConsultationBeds, {
+    query: {
+      consultation: props.consultationId,
+    },
+    onResponse: ({ res, data }) => {
+      if (res?.ok && data) {
+        setBedAssignmentStartDate(data.results[0].created_date);
+      } else {
+        Notification.Error({
+          msg: "Something went wrong: ",
+        });
+      }
+    },
+  });
```

Comment:
You don't need a seperate `useState` for `bedAssignmentStartDate`. Avoid using `useStates` when you can approach it without one. `useQuery` already manages a state of the data for you.

```suggestion
  const { data } = useQuery(routes.listConsultationBeds, {
    query: { consultation: props.consultationId },
  });
  const bedAssignmentStartDate = data?.results[0].created_date ?? "";
```